easybuild/easyconfigs/n/Nipype/Nipype-1.6.0-foss-2020b.eb

```diff
@@ -0,0 +1,87 @@
+easyblock = 'PythonBundle'
+
+name = 'Nipype'
+version = '1.6.0'
+
+homepage = 'https://nipype.readthedocs.io'
+description = """Nipype is a Python project that provides a uniform interface to existing neuroimaging software and
+ facilitates interaction between these packages within a single workflow."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('lxml', '4.6.2'),
+    ('networkx', '2.5'),
+    ('NiBabel', '3.2.1'),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
```

Comment:
To fix `AssertionError: 'source_urls' should not be defined in exts_default_options when using the default value in Nipype-1.6.0-foss-2020b.eb`:
```suggestion
```